scanpipe/pipes/rootfs.py

```diff
@@ -336,3 +340,94 @@ def tag_uninteresting_codebase_resources(project):
 
     qs = project.codebaseresources.no_status()
     qs.filter(lookups).update(status="ignored-not-interesting")
+
+
+def tag_ignorable_codebase_resources(project):
+    """
+    Using the glob patterns from commoncode.ignore of ignorable files/directories,
+    tag codebase resources from `project` if their paths match an ignorable pattern.
+    """
+    lookups = Q()
+    for pattern in default_ignores.keys():
+        # Translate glob pattern to regex
+        translated_pattern = fnmatch.translate(pattern)
+        # PostgreSQL does not like parts of Python regex
+        if translated_pattern.startswith("(?s"):
+            translated_pattern = translated_pattern.replace("(?s", "(?")
+        lookups |= Q(rootfs_path__icontains=pattern)
+        lookups |= Q(rootfs_path__iregex=translated_pattern)
+
+    qs = project.codebaseresources.no_status()
+    qs.filter(lookups).update(status="ignored-default-ignores")
+
+
+def tag_data_files_with_no_clues(project):
+    """
+    Tags CodebaseResources that have a file type of `data` and no detected clues
+    to be uninteresting.
+    """
+    lookup = Q(
+        file_type="data",
+        copyrights=[],
+        holders=[],
+        authors=[],
+        licenses=[],
+        license_expressions=[],
+        emails=[],
+        urls=[],
+    )
+
+    qs = project.codebaseresources
+    qs.filter(lookup).update(status="ignored-data-file-no-clues")
+
+
+def tag_media_files_as_uninteresting(project):
+    """
+    Tags CodebaseResources that are media files to be uninteresting.
+
+    `mimes` and `types` are taken from TypeCode:
+    https://github.com/nexB/typecode/blob/main/src/typecode/contenttype.py#L528
+    """
+    mimes = (
```

Comment:
This should be directly imported from `typecode`.
It would require to make https://github.com/nexB/typecode/blob/main/src/typecode/contenttype.py#L528 available as a module variable.
In the short term, we can keep it as-is and enter a ticket on the `typecode` side.

Reply:
@tdruez On the other hand, what if we added the `is_media` field from the license scan to the `CodebaseResource` model?

Reply:
Sure, that sounds good.

Reply:
I've added the `is_media` field to `CodebaseResource`, removed the part that excludes `is_media` from `unsupported_fields` in `scanpipe.pipes.scancode.get_resource_info`, and updated the tests results.